plugins/Installation/lang/en.json

```diff
@@ -158,6 +158,7 @@
         "EmailPrivacyNotice": "Your email address will only be used to send you the newsletter. It is shared with Mad Mimi to do so, but the third-party provider may change. We will not share your email with anyone else or use your it for any other purpose. Unsubscribe at any time. The %1$sprivacy policy%2$s has more info.",
         "PerformanceSettingsDesc1": "Your Matomo is set up and ready to track and report on your website's traffic. Set up %1$sCLI archiving%2$s if you find it to be slow. This generates reports in the background, rather than on demand.",
         "PerformanceSettingsDesc2": "This requires adding a Matomo command to Cron which can't be done automatically by the installer. %1$sRead our FAQ to learn to set it up yourself.%2$s",
-        "MatomoHttpRequestConfigInfo": "The option force_matomo_http_request in the general config is turned on, we highly recommend switching it off for security reasons. For more details please read our %1$sFAQ%2$s"
+        "MatomoHttpRequestConfigInfo": "The option <code>force_matomo_http_request</code> in the general config is turned on, we highly recommend switching it off for security reasons. For more details please read %1$show to disable https for matomo.org and api.matomo org.requests%2$s",
+        "MatomoHttpsNotSupported": "Could not initialize HTTPS. Connections to matomo.org use HTTPS by default for security. Please make sure that HTTPS works on your environment or enable the option `force_matomo_http_request`. For more details please read %1$show to disable https for matomo.org and api.matomo.org requests%2$s"
```

Comment:
Actually the config setting forces http connections to plugins.matomo.org as well. Maybe we should simply write `how to disable https for requests to matomo.org`

Reply:
Agreed, the various matomo.org subdomains aren't relevant to users. I've also adjusted the FAQ to match this (but not the FAQ url as the it's referenced in the previous release)